### PR TITLE
[DOCS] #5000 – Add rendering docs to release todo

### PR DESCRIPTION
**Pull request by fantasya-pbem**
Backport from master, see #5045, including various other consistency changes.

### CODE REVIEW

**Comment by ThomasWaldmann** [comment]
The diff doesn't look like a clean backport.

**Comment by fantasya-pbem** [comment]
I took over all the clean-up changes that happend in this file in master, but didn't touch the python versions.

File: docs/development.rst [hunk 1]
```diff
@@ -5,9 +5,9 @@
 Development
 ===========
 
-This chapter will get you started with |project_name| development.
+This chapter will get you started with Borg development.
```
Comment (ThomasWaldmann) [inline docs/development.rst]: are this kind of changes present in other 1.1-maint docs files? if not, it should not be here either.

Comment (fantasya-pbem) [reply docs/development.rst]: You're right, the |project_name| thing is used everywhere. I will undo that change.

**Comment by ThomasWaldmann** [comment]
Thanks!

**Pull request merged**